### ISSUE TITLE
Error when loading stream

### ISSUE THREAD

**Issue by Vuccappella** [crackededed/Xtra#359]
Anytime i click on a stream i get a black screen and an error saying error when loading stream, any ideas how to fix?

**Comment by Vuccappella** [comment]
> It's probably because you've the option **Hide ad segments** turned on and you're still using the outdated TTV.LOL proxy to hide them, the proxy was probably offline that's why they didn't load.
> 
> I use this settings:
> 
> Stream proxy -> Custom url -> Custom proxy url: `https://lb-eu.cdn-perfprod.com/live/$channel?allow_source=true&allow_audio_only=true&fast_bread=true`
> 
> and
> 
> HTTP proxy settings:
> 
> * Host -> `firefox.api.cdn-perfprod.com`
> * Port -> `2023`
> 
> Give them a try and report back.

Yhanks for the reply, i havent actually adjusted any of these settings but went ahead and disabled the option under Player -- Use Ad  Blocker ( Uses the TTV.lol api to load streams) but streams still error put the same way.

For the aditional settings you noted, i cant dind them in my settings , on the player wettings i have playback acces tokeheader option but i cant set http proxy there and api settings but i cant se these things there either, where should i navigate?

**Comment by AitoZ** [comment]
Make sure you're on the latest version **2.24.2**, open the app -> top right (3 dots) -> Settings then scroll all the way down and after that a bit up. There you should see the settings I mentioned before, if that doesn't help wait for an answer from @crackededed.

**Comment by Vuccappella** [comment]
Updating gelped, for some reason i was on 2.03 and fdroid couldnt fetch a newer version for me so i manually downloaded the latest apk from here and that fixed the issue., is there a point for using the cuatom proxy settings? Whats the difference between them and the default?

**Comment by AitoZ** [comment]
The default proxy (TTV.LOL) has been a hit or miss for me, most of the time it was offline and I had to turn off the option **Hide ad segments** to be able to watch the streams. The custom ones have been very stable and most of the ads get blocked, sometimes you get a mid-roll ad but that doesn't happen often. I got them from [here](https://github.com/younesaassila/ttv-lol-pro/discussions/37) and [here](https://github.com/younesaassila/ttv-lol-pro/discussions/130) if you want to take a look, I hope I was able to help.
